modules/layers/src/arc-layer/arc-layer.js

```diff
@@ -58,15 +58,11 @@ export default class ArcLayer extends Layer {
     attributeManager.addInstanced({
       instancePositions: {
         size: 4,
+        doublePrecision: this.use64bitPositions(),
```

Comment:
This option determines whether the attribute is auto-packed into a `Float32Array` or a `Float64Array`. This is important because we don't want additional perf overhead for AVS (meter offset) use cases. Since the switch is dependent on `layer.props.coordinateSystem`, I find it the cleanest to define it once in initialization, versus making the `Attribute` class understand the layer context.